pkgs/by-name/in/inngest/package.nix

```diff
@@ -0,0 +1,88 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  stdenv,
+  pnpm_8,
+  nodejs_20,
+}:
+let
+  version = "1.6.3";
+  shortCommit = "9b27357";
+
+  src = fetchFromGitHub {
+    owner = "inngest";
+    repo = "inngest";
+    tag = "v${version}";
+    hash = "sha256-6K8Tshdg0WjQ2yWMY6bLKJOzY0H+vwrLyLQvN2D4758=";
+  };
+
+  pnpm = pnpm_8.override {
+    version = "8.15.8";
+    hash = "sha256-aR/hdu6pqKgN8g5JdvPftEoEhBzriFY4/iomF0+B5l4=";
+    nodejs = nodejs_20;
+  };
+
+  ui = stdenv.mkDerivation (finalAttrs: {
+    inherit version src;
+    pname = "ui";
+
+    nativeBuildInputs = [
+      (pnpm.configHook.override { inherit pnpm; })
+      pnpm
+      nodejs_20
+    ];
+
+    buildPhase = ''
+      cd ui/apps/dev-server-ui && pnpm build
+    '';
+
+    installPhase = ''
+      mkdir -p $out
+      cp -r dist $out
+      cp -r .next/routes-manifest.json $out/dist
+    '';
+
+    pnpmWorkspaces = [ "dev-server-ui" ];
+
+    pnpmDeps = pnpm.fetchDeps.override { inherit pnpm; } {
+      inherit (finalAttrs) pname pnpmWorkspaces;
+      inherit version src;
+      hash = "sha256-WGZK/93WJiXryvLyGLadJ/QlHxK1p8PM53JVOXL6uq0=";
+      sourceRoot = "${finalAttrs.src.name}/ui";
+    };
+    pnpmRoot = "ui";
+  });
+in
+buildGoModule {
+  inherit version src;
+  pname = "inngest";
+
+  vendorHash = null;
+
+  preBuild = ''
+    go generate ./...
+    cp -r ${ui}/dist/* ./pkg/devserver/static
+  '';
+
+  postInstall = ''
+    mv $out/bin/cmd $out/bin/inngest
+  '';
+
+  ldflags = [
+    "-s -w"
+    "-X github.com/inngest/inngest/pkg/inngest/version.Version=${version}"
+    "-X github.com/inngest/inngest/pkg/inngest/version.Hash=${shortCommit}"
+  ];
+
+  env.CGO_ENABLED = 0;
```

Comment:
Is there any particular reason why `CGO_ENABLED` has to be set to 0? If so, please add a comment explaining why

Reply:
Fair enough - I would still recommend adding a comment (+ maybe a link to official docs)